studyspaces/SwiftUIView.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  SwiftUIView.swift
+//  studyspaces
+//
+//  Created by Josh Cho on 11/12/24.
+//
+import MapKit
+import SwiftUI
+
+
+extension SwiftUIView {
```

Comment:
this could use the enum paradigm I showed two weeks ago
```
enum actions {
case clearRoute
case searchPlace
...
}

Then you would put this functionality in a ViewModel
```